NOTICE.md

```diff
@@ -71,3 +71,33 @@ The project maintains the following source code repositories:
 ### jquery.matchHeight.js (^0.7.2)
 
 * License: The MIT License (MIT)
+
+### React (^16.13.1)
+
+* License: The MIT License (MIT)
+
+### prop-types (^15.7.2)
+* License: The MIT License (MIT)
+
+### @babel/preset-env (^7.11.5)
+
+* License: The MIT License (MIT)
+
+### @babel/preset-react (^7.10.4)
+* License: The MIT License (MIT)
+
+### @testing-library/jest-dom (^5.11.4)
```

Comment:
The testing libs aren't bundled and are dev deps so they don't have to be strictly included

Reply:
Made some adjustments, removed the dev dependency of testing libs. 

But noticed that compile libs should be in prod dependency.